static/app/components/eventsTable/eventsTable.tsx

```diff
@@ -40,6 +49,7 @@ class EventsTable extends Component<Props> {
               groupId={groupId}
               tagList={tagList}
               hasUser={hasUser}
+              showReplayColumn={Boolean(replayColumn)}
```

Comment:
should cast to boolean when the var is defined. otherwise the `&&` will cause someone a problem if it renders `0` into the dom

Reply:
Good catch. I originally was using the column for the name but decided I could just make the name literal